.eslintrc.js

```diff
@@ -1,25 +1,42 @@
 module.exports = {
   extends: 'airbnb',
+  env: { browser: true },
   rules: {
+    // ESLint ----------------------------------
+    'arrow-parens': ['error', 'always'],
```

Comment:
I think it can clear things up for more complicated generator functions. 
https://github.com/1-liners/1-liners/blob/master/module/composeAll.js

Nevertheless [Airbnb uses, arrow-parens: "as-needed", { "requireForBlockBody": true }](https://github.com/airbnb/javascript#arrows--one-arg-parens), I'll go for fewer eslintrc and revert this rule and its changes.